pybossa/view/projects.py

```diff
@@ -1873,7 +1873,10 @@ def bulk_update_assign_worker(short_name):
                     if remove_user_email in assign_user:
                         assign_user.remove(remove_user_email)
 
-                user_pref["assign_user"] = assign_user
+                if assign_user:
+                    user_pref["assign_user"] = assign_user
+                elif "assign_user" in user_pref:
```

Comment:
Is this delete key logic covered in the unit test?

Reply:
Unit tests added, thank you. :)